x11-wm/e16/e16-1.0.20.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Enlightenment Window Manager (E16)"
+HOMEPAGE="https://www.enlightenment.org https://sourceforge.net/projects/enlightenment/"
+SRC_URI="mirror://sourceforge/enlightenment/${P}.tar.xz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE="audiofile container dbus debug +dialogs doc examples gnome
+libhack modules nls opengl +pango sndfile sound +themes xcomposite
++xft xi2 xinerama xpresent +xrandr +xrender +xsm +xsync zoom"
+
+BDEPEND="
+	nls? ( sys-devel/gettext )
+	virtual/pkgconfig
+"
+CDEPEND="
+	dbus? ( sys-apps/dbus )
+	doc? ( app-doc/e16-docs )
+	nls? ( virtual/libintl )
+	opengl? ( media-libs/glu media-libs/mesa )
+	pango? ( dev-libs/glib:2 x11-libs/pango[X] )
+	sound? (
+		|| ( media-sound/pulseaudio media-sound/apulse[sdk] )
+		sndfile? ( media-libs/libsndfile )
+		audiofile? ( media-libs/audiofile )
+	)
+	xcomposite? (
+		x11-libs/libXcomposite
+		x11-libs/libXdamage
+		x11-libs/libXfixes
+	)
+	xft? ( x11-libs/libXft )
+	xinerama? ( x11-libs/libXinerama )
+	xpresent? ( x11-libs/libXpresent )
+	xrandr? ( x11-libs/libXrandr )
+	xrender? ( x11-libs/libXrender )
+	xsm? ( x11-libs/libICE x11-libs/libSM )
+	zoom? ( x11-libs/libXxf86vm )
+	=media-libs/freetype-2*
```

Comment:
- `media-libs/freetype` has a slot, so just depend on slot `2` instead
- optional deps come below non-optional deps, please move this block accordingly